Bikes/Bikes.Application.Contracts/Dtos/BikeDto.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Bikes.Application.Contracts.Dtos;
+
+/// <summary>
+/// DTO for reading a bicycle
+/// </summary>
+public record BikeDto
+{
+    /// <summary>
+    /// Unique bicycle identifier
+    /// </summary>
+    public int Id { get; init; }
+
+    /// <summary>
+    /// Serial number
+    /// </summary>
+    public string SerialNumber { get; init; }
+
+    /// <summary>
+    /// Bicycle color
+    /// </summary>
+    public string Color { get; init; }
+
+    /// <summary>
+    /// Bicycle model
+    /// </summary>
+    public BikeModelDto Model { get; init; }
+
+    /// <summary>
+    /// DTO constructor
+    /// </summary>
+    public BikeDto(int id, string serialNumber, string color, BikeModelDto model)
+    {
+        Id = id;
+        SerialNumber = serialNumber;
+        Color = color;
+        Model = model;
+    }
+}
```

Comment:
Данный код можно сократить воспользовавшись Primary constructor:
```suggestion
public record BikeDto(int Id, string SerialNumber, string Color, BikeModelDto Model);
```
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record
Компилятор автоматически сгенерирует:
1. Конструктор и Деконструктор;
2. Свойства init-only;
4. Equals, GetHashCode, ToString